### PR TITLE
panc development: add BTrace example code and usage

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by stdweird** [comment]
Was instrumental in solving https://github.com/quattor/pan/issues/163

File: _posts/2016-06-20-panc-development.md [hunk 1]
```diff
@@ -271,3 +271,181 @@ built project can be executed on the command line as follows:
...
+        java.net.URLClassLoader.findClass(URLClassLoader.java:381)
+        java.lang.ClassLoader.loadClass(ClassLoader.java:424)
+
+
+
+[btrace]: https://github.com/btraceio/btrace
+
+### Usage
+
+Get [latest BTrace release] and unpack it.
```
Comment (ned21) [inline _posts/2016-06-20-panc-development.md]: This doesn't seem to render correctly in the preview?

File: _posts/2016-06-20-panc-development.md [hunk 2]
```diff
@@ -271,3 +271,181 @@ built project can be executed on the command line as follows:
...
+(`AllCall3`, `AllLines`, `AllMethods` and `OnThrow`).
+
+The output is very verbose and contains 3 things:
+* Every linenumber in any used `quattor.org` class, e.g.
+
+    org.quattor.pan.tasks.FinalResult.resolveAllDependencies:97
+
+* Every method call in any used `quattor.org` class, e.g.
+
+    entered org.quattor.pan.type.AliasType.verifySubtypesDefined
```
Comment (ned21) [inline _posts/2016-06-20-panc-development.md]: This doesn't to render as pre-formatted text in the GH preview?

Comment (jrha) [reply _posts/2016-06-20-panc-development.md]: Triple back-ticks are the most robust way to ensure pre-formatted code blocks work across multiple markdown engines, indentation *mostly* works.

**Comment by stdweird** [comment]
@ned21 should be better now

File: _posts/2016-06-20-panc-development.md (hunk 1 above)
Comment (ned21) [reply _posts/2016-06-20-panc-development.md]: Should this line be ```[latest BTrace release][btrace_latest]``` ?

**Comment by ned21** [comment]
One more link to fix I think. Looks like #216 is causing lots of errors (non-fatal?) in travis?

**Comment by stdweird** [comment]
@ned21 fixed. and i disabled the spellchecker and opened #229 for followup

**Comment by jrha** [comment]
@stdweird #230 should fix the spellchecker problems

**Comment by stdweird** [comment]
@jrha any tips how to fix this? add more items to the dictionary?

**Comment by jrha** [comment]
@stdweird wrap things like filenames and words that are code-like things in backticks .e.g. `WriteOutput`, the spellchecker will then ignore them. After that you can add words to the dictionary.

**Comment by jrha** [comment]
The remaining test failures should be fixed by https://github.com/stfc/markdown-spellchecker/pull/10

**Comment by jouvin** [comment]
@stdweird any chance to see this PR fixed so that it could be merged?

**Comment by stdweird** [comment]
@jouvin done. totally forgot about this PR....

**Comment by jouvin** [comment]
@stdweird this documentation seems important for anybody who wants to contribute to panc and I don't like it is in the middle of the posts... I'd suggest to move it to the Documentation->Development menu and add in the front-matter section a `redirect_from` to ensure that the old post URL is still valid.

**Comment by jouvin** [comment]
@stdweird almost good 😄 I suggest to use a shorter text for 'menu' attribute, like `panc` or `Pan compiler` as it is used in the left pane.

Not clear why the spellchecker is unhappy: it seems to have a problem with the renamed file... let 's forget about it.

**Comment by stdweird** [comment]
@jouvin thanks for the PR. test is ok now

**Comment by jouvin** [comment]
I am not completely sure that the modification does what I want but let follow this in later PRs...

**Pull request merged**